README.md

```diff
@@ -143,6 +143,79 @@ Priority of configuration (high to low):
 3. Defined configuration file
 4. Default configuration file
 
+Every value in the config file can be set through environment variables. 
+
+You can set a token for the http loader:
+
+```bash
+export SPARROW_LOADER_HTTP_TOKEN="Bearer xxxxxx"
+```
+
+Or for any other config attribute:
+```bash
+export SPARROW_ANY_OTHER_OPTION="Some value"
+```
+
+Just write out the path to the attribute, delimited by `_`.
+
+
+Example configuration:
+```yaml
+# DNS sparrow is exposed on 
+name: sparrow.example.com
+# Selects and configures a loader for continuosly fetching the configuration at runtime
+loader:
+    # defines which loader to use. Options: "file | http" 
+    type: http  
+    # the interval in which sparrow tries to fetch a new configuration
+    interval: 30s
+    # config specific to the http loader
+    http: 
+        # The url where the config is located
+        url: https://myconfig.example.com/config.yaml
+        # This token is passed in the Authorization header, when refreshing the config
+        token: Bearer xxxxxxx
```

Comment:
Remove the Bearer since it'd be duplicated if new users keep it and just but their token instead of the placeholder.

The risk is that the authorization header would look like this if they configure it wrong:
```
Authorization: Bearer Bearer xxxx
```